package/yast2-pam.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-pam
-Version:        4.1.0
+Version:        4.1.1
```

Comment:
This is master only, there will be no backport to SP1, right? In that case the version should be bumped to `4.2.0`.

Reply:
OK, fine with me. But then y-users will also get the 4.2.0 version.